src/internal/packager/images/push.go

```diff
@@ -50,6 +51,7 @@ func (i *ImageConfig) PushToZarfRegistry() error {
 
 	httpTransport := http.DefaultTransport.(*http.Transport).Clone()
 	httpTransport.TLSClientConfig.InsecureSkipVerify = i.Insecure
+	httpTransport.ResponseHeaderTimeout = 10 * time.Second
```

Comment:
for faster experimentation, should we expose this as a config option?

99% of the time 10s is prob fine, but could be weird cases w/ satcom connections

Reply:
After doing a bit more thinking / research I think this would be much better explored in a dedicated issue: https://github.com/defenseunicorns/zarf/issues/2331

It should ideally be something we can apply globally and for sat com connections there are other hard-coded timeouts in the default transport as well (10s is the same as the TLS handshake timeout and should be a reasonably conservative choice for an image registry to return its headers)